apps/test/unit/lib/kits/maker/ThermometerTest.js

```diff
@@ -62,3 +66,74 @@ describe('Thermometer', function() {
     }
   });
 });
+
+describe('MicroBitThermometer', function() {
+  let boardClient;
+  let thermometer;
+
+  beforeEach(() => {
+    boardClient = new MicrobitStubBoard();
+    thermometer = new MicroBitThermometer({mb: boardClient});
+  });
+
+  it(`attributes are readonly`, () => {
+    let attributes = ['raw', 'celsius', 'fahrenheit', 'C', 'F'];
+    let desc;
+
+    attributes.forEach(attr => {
+      desc = Object.getOwnPropertyDescriptor(thermometer, attr);
```

Comment:
nit: can you replace `desc` with `description`? `desc` is often used as a shorthand for `descending` as well. It's nice to disambiguate.